capstone-backend/src/main/java/com/google/sps/data/BookList.java

```diff
@@ -17,77 +17,77 @@
  */
 public final class BookList {
 
-  private Long id;
-  private Long userID;
+  private String id;
+  private String userID;
   private String bookshelfID;
-  private Collection<Long> collaboratorsIDs;
+  private Collection<String> collaboratorsIDs;
 
-  public BookList(Long userID, String bookshelfID, Collection<Long> collaboratorsIDs, Long id) {
+  public BookList(String userID, String bookshelfID, Collection<String> collaboratorsIDs, String id) {
```

Comment:
Can you make the id the first parameter in the constructor since it's usually the 'primary' key?

Reply:
Ok